### ISSUE TITLE
Scrollbar for UUID selector dropdown

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#301]
## Description
The dropdown in the Editor for STB is already too long, with 23 products in the list and many more to come. A scrollbar is needed to traverse through this list, and a height should be set so the dropdown isn't flowing right off of the screen.
![image](https://github.com/ramp4-pcar4/storylines-editor/assets/5294596/08dc2629-ef74-4ced-a632-c5e1690cba1e)

## Alternatives considered
Open to ideas!

**Comment by dan-bowerman** [comment]
As discussed in our 6/6 meeting, Matt is going to investigate storing and populating the Storylines title (we probably want to support english and french titles depending on the language of the user.

UUID - Title should be fine for now. The title can be truncated if their is not enough horizontal space. The labels in the load form can probably be tightened up to provide more real estate as well.
